libs/perun/dialogs/src/lib/password-reset-dialog/password-reset-dialog.component.html

```diff
@@ -0,0 +1,60 @@
+<ng-template #spinner>
+  <perun-web-apps-loading-dialog></perun-web-apps-loading-dialog>
+</ng-template>
+<div class="user-theme position-relative">
+  <div class="title-with-icon">
+    <h1 mat-dialog-title>
+      {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.TITLE' | translate}}
+    </h1>
+  </div>
+  <div class="dialog-container" mat-dialog-content>
+    <p *ngIf="getLogin()" class="subtitle">
+      <em
+        >{{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.LOGIN' | translate}}:
+        {{getLogin()}}</em
+      >
+    </p>
+    <p *ngIf="getNamespace()" class="subtitle">
+      <em
+        >{{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.NAMESPACE' | translate}}:
+        {{getNamespace()}}</em
+      >
+    </p>
+    <perun-web-apps-password-form
+      class="ml-3 me-3"
+      *ngIf="!success && !loading && !error"
+      [formGroup]="newPasswdForm"
+      [namespace]="getNamespace()"
+      [language]="language">
+    </perun-web-apps-password-form>
+    <div *ngIf="success && !loading" class="mt-5">
+      <perun-web-apps-alert alert_type="success">
+        {{successMsg}}
+      </perun-web-apps-alert>
+    </div>
+
+    <div *ngIf="error && !loading" class="mt-5">
+      <perun-web-apps-alert alert_type="error">
+        {{errorMsg}}
+      </perun-web-apps-alert>
+    </div>
+  </div>
```

Comment:
There is no need to show any success / error message inside the dialog. Show a notification on success, error will be shown automatically if it occurs.